apps/ui/src/i18n/locale/en/enSection.json

```
@@ -196,6 +196,11 @@
 			"description": "Are you sure you want to submit Application: PCGL-{{id}}? If so, the application will be locked for editing and the institutional Representative will be notified to begin the review process.",
 			"submit": "Proceed with submission",
 			"cancel": "Cancel"
+		},
+		"notifications": {
+			"submitApplicationSuccess": "Your application has been submitted successfully.",
```

Just a small change to align it with the mockup text:
```suggestion
			"submitApplicationSuccess": "Your application has been submitted for review.",
```